airbyte-connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -185,21 +179,16 @@ components:
         headers:
           type: object
           description: The headers of the HTTP response, if any
-    # --- Commenting out for now since they do not work with our orval openapi client generator ---
-    # ConnectorManifest:
-    #   $ref: ../../../../airbyte-cdk/python/airbyte_cdk/sources/declarative/config_component_schema.json
-    # AirbyteProtocol:
-    #   $ref: ../../../../airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml
```

Comment:
Removing AirbyteProtocol from this spec, because it still has issues being pulled in which aren't the fault of orval, but more an incompatibility between JSON schema features that the AirbyteProtocol is using and the version of OpenAPI that most generator tools still support.

This means we are in solution number 2 as described [here](https://github.com/airbytehq/airbyte/issues/18258) which is a totally satisfactory position to be in.